indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamSamplerSpec.java

```diff
@@ -102,7 +106,7 @@ public SamplerResponse sample()
           "[spec.ioConfig.inputFormat] is required"
       );
     }
-
+    inputSource.validateAllowDenyPrefixList(securityConfig);
```

Comment:
nit: this doesn't look harm because neither `RecordSupplierInputSource` nor `FirehoseFactoryToInputSourceAdaptor` do anything in `validateAllowDenyPrefixList()`. However, it looks confusing since `validateAllowDenyPrefixList()` is not called in `RecordSupplier`. I would suggest to delete this line and add it back when we truly support this security config for stream ingestion.

Reply:
done.